File: Server/src/main/java/com/api/server/foodb/SimulateContent.java
package com.api.server.foodb;

import jakarta.persistence.*;
import jakarta.validation.constraints.NotNull;
import jakarta.validation.constraints.Size;
import lombok.Getter;
import lombok.Setter;

import java.math.BigDecimal;
import java.time.Instant;

@Getter
@Setter
@Entity
@Table(name = "simulate_contents")
public class SimulateContent {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false)
    private Integer id;

    @Column(name = "source_id")
    private Integer sourceId;

    @Size(max = 255)
    @Column(name = "source_type")
    private String sourceType;

    @Column(name = "food_id")
    private Integer foodId;

    @Column(name = "orig_food_id")
    private Integer origFoodId;

    @Size(max = 255)
    @Column(name = "orig_food_part")
    private String origFoodPart;

    @Size(max = 255)
    @Column(name = "orig_source_id")
    private String origSourceId;

    @Size(max = 255)
    @Column(name = "orig_source_name")
    private String origSourceName;

    @Size(max = 255)
    @Column(name = "orig_citation")
    private String origCitation;

    @Size(max = 255)
    @Column(name = "citation")
    private String citation;

    @Size(max = 255)
    @Column(name = "citation_type")
    private String citationType;

    @NotNull
    @Column(name = "created_at", nullable = false)
    private Instant createdAt;

    @NotNull
    @Column(name = "updated_at", nullable = false)
    private Instant updatedAt;

    @Column(name = "orig_min", precision = 15, scale = 9)
    private BigDecimal origMin;

    @Column(name = "orig_max", precision = 15, scale = 9)
    private BigDecimal origMax;

    @Size(max = 255)
    @Column(name = "orig_unit")
    private String origUnit;

    @Size(max = 255)
    @Column(name = "orig_food_common_name")
    private String origFoodCommonName;

    @Size(max = 255)
    @Column(name = "orig_food_scientific_name")
    private String origFoodScientificName;

    @Column(name = "orig_content", precision = 15, scale = 9)
    private BigDecimal origContent;

    @Size(max = 255)
    @Column(name = "creater_id")
    private String createrId;

    @Size(max = 255)
    @Column(name = "updater_id")
    private String updaterId;

    @Size(max = 255)
    @Column(name = "orig_method")
    private String origMethod;

    @Size(max = 255)
    @Column(name = "orig_unit_expression")
    private String origUnitExpression;

    @Column(name = "standard_content", precision = 10)
    private BigDecimal standardContent;

    @Size(max = 255)
    @Column(name = "preparation_type")
    private String preparationType;

    @Size(max = 255)
    @NotNull
    @Column(name = "food_type", nullable = false)
    private String foodType;

}

File: DotNetServer/DotNetServer/Core/Entities/Foodb/SimulateContent.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore;

namespace DotNetServer.Core.Entities.Foodb;

[Table("simulate_contents")]
[Index("SourceId", "SourceType", "FoodId", Name = "index_simulate_contents_on_source_id_and_source_type_and_food_id")]
public partial class SimulateContent
{
    [Key] [Column("id")] public int Id { get; set; }

    [Column("source_id")] public int? SourceId { get; set; }

    [Column("source_type")] public string? SourceType { get; set; }

    [Column("food_id")] public int? FoodId { get; set; }

    [Column("orig_food_id")] public int? OrigFoodId { get; set; }

    [Column("orig_food_part")]
    [StringLength(255)]
    public string? OrigFoodPart { get; set; }

    [Column("orig_source_id")]
    [StringLength(255)]
    public string? OrigSourceId { get; set; }

    [Column("orig_source_name")]
    [StringLength(255)]
    public string? OrigSourceName { get; set; }

    [Column("orig_citation")]
    [StringLength(255)]
    public string? OrigCitation { get; set; }

    [Column("citation")]
    [StringLength(255)]
    public string? Citation { get; set; }

    [Column("citation_type")]
    [StringLength(255)]
    public string? CitationType { get; set; }

    [Column("created_at", TypeName = "datetime")]
    public DateTime CreatedAt { get; set; }

    [Column("updated_at", TypeName = "datetime")]
    public DateTime UpdatedAt { get; set; }

    [Column("orig_min")]
    [Precision(15, 9)]
    public decimal? OrigMin { get; set; }

    [Column("orig_max")]
    [Precision(15, 9)]
    public decimal? OrigMax { get; set; }

    [Column("orig_unit")]
    [StringLength(255)]
    public string? OrigUnit { get; set; }

    [Column("orig_food_common_name")]
    [StringLength(255)]
    public string? OrigFoodCommonName { get; set; }

    [Column("orig_food_scientific_name")]
    [StringLength(255)]
    public string? OrigFoodScientificName { get; set; }

    [Column("orig_content")]
    [Precision(15, 9)]
    public decimal? OrigContent { get; set; }

    [Column("creater_id")]
    [StringLength(255)]
    public string? CreaterId { get; set; }

    [Column("updater_id")]
    [StringLength(255)]
    public string? UpdaterId { get; set; }

    [Column("orig_method")]
    [StringLength(255)]
    public string? OrigMethod { get; set; }

    [Column("orig_unit_expression")]
    [StringLength(255)]
    public string? OrigUnitExpression { get; set; }

    [Column("standard_content")]
    [Precision(10, 0)]
    public decimal? StandardContent { get; set; }

    [Column("preparation_type")]
    [StringLength(255)]
    public string? PreparationType { get; set; }

    [Column("food_type")]
    [StringLength(255)]
    public string FoodType { get; set; } = null!;
}